### ISSUE TITLE
Calibre-web broken after update 

### ISSUE THREAD

**Issue by Cloudhidden** [janeczku/calibre-web#2290]
**Describe the bug/problem**
After updating calibre-web via web interface to the latest version released on 1/22 my installation is absolutely broken and I cannot access the main page. I get "Connection Refused"

**To Reproduce**
Log into calibre-web
Go to settings and update
web interface is no longer accessible. Throws error "Connection refused"

**Logfile**

pi@Pi3-Library:~ sudo cat nohup.out
python: can't open file 'cps.py': [Errno 2] No such file or directory
pi@Pi3-Library:~ $ python3 -m cps
/usr/bin/python3: No module named cps

I can't find calibre-web.log for some reason. It looks like calibre-web was removed after the update.

**Expected behavior**
I expect the update not to break my installation that had been running fine for a year straight. 

**Screenshots**
![image](https://user-images.githubusercontent.com/82607052/151048127-c02869bb-3deb-4ef9-b94a-0115e858c54a.png)

**Environment (please complete the following information):**
 - OS: Raspberry Pi OS
 - Python version: 3.x
 - Calibre-Web version: I don't know. I can't check cause I can't connect to it.
 - Special Hardware: Raspberry Pi 3
 - Browser: Vivaldi, 5.0.2497.48

**Additional context**
Calibre-web was installed manually from the instructions in this github. No docker images.

**Comment by OzzieIsaacs** [comment]
It's true, looks like its gone. Please check if the files are there, if not, try to reinstall it

**Comment by OzzieIsaacs** [comment]
I'm regulary using the in program updater, without any problems. I have no idea what went wrong

**Comment by Cloudhidden** [comment]
I'm still looking but it looks like my entire installation is gone. Wtf.

No process using port 8083. Nothing in any sys log. 

This is troublesome. How am I to update my installation in the future? Is this going to happen again?

**Comment by OzzieIsaacs** [comment]
No idea

**Comment by Cloudhidden** [comment]
I found the calibre-web.log in a completely different folder than where it should be and I found this

` INFO {cps:128} *** "flask-WTF" is needed for calibre-web to run. Please install it using pip: "pip install flask-WTF" ***`

I noticed this issue https://github.com/janeczku/calibre-web/issues/2161

So I installed flask-wtf with 

sudo pip3 install flast-WTF

Install was successful but when restarting calibre-web service I get:

pi@Pi3-Library:~ $ systemctl restart calibreweb
Failed to restart calibreweb.service: Unit calibreweb.service not found.

**Comment by OzzieIsaacs** [comment]
> Unit calibreweb.service not found

This is not a calibre-web file, it's the service file from systemd. I suggest to get calibre-web back running from terminal, and afterwards fix the service file (maybe it's just the changed folder location).

**Comment by Cloudhidden** [comment]
I was able to salvage the installation by finding the existing install folder and running the command

`pip3 install --target vendor -r requirements.txt`

I guess all the dependencies really were wiped out because they were all reinstalled.

I ran `python3 cps.py` to confirm and I can log into the web interface again.

I checked the update section in the settings and there are new updates to install

![image](https://user-images.githubusercontent.com/82607052/151053501-0a1b209a-7d8f-40b8-8bc4-fc1424ccc8ae.png)

But I'm not confident running the updater again. The issue is somewhat resolved as long as I don't run the updater again so I guess I'm staying on version 0.6.17 indefinitely.
